src/dmd/frontend.h

```diff
@@ -4991,6 +4991,24 @@ enum class CPU
     native = 12,
 };
 
+struct BucketUsageInfo final
```

Comment:
These definitions still have to be put in a header for gdc and ldc to make use of them.

Reply:
so they can't jut live in `frontend.h` ?

Reply:
Nope, no one uses this, it's just a device to force people to think about headers.

Reply:
I added aa.h